pinot-common/src/main/java/org/apache/pinot/common/utils/DataTable.java

```diff
@@ -44,6 +44,7 @@
   String REQUEST_ID_METADATA_KEY = "requestId";
   String NUM_RESIZES_METADATA_KEY = "numResizes";
   String RESIZE_TIME_MS_METADATA_KEY = "resizeTimeMs";
+  String THREAD_TIME_MS = "threadTimeMs";
```

Comment:
Change it to micro seconds please

Reply:
Why should this be changed to microseconds?
The jmx api documentation says it returns nanosecond precision (but may not be necessarily accurate). But we can't treat it as microseconds.

For example, if the returned value is 
1000000. It is 1000 microseconds. If we treat the returned value as microseconds, then we are increasing the value by 1000 times making it 1000ms.

Reply:
done